docs/interpreter/livy.md

```diff
@@ -6,8 +6,12 @@ group: manual
 ---
 {% include JB/setup %}
 
-## Livy Interpreter for Apache Zeppelin
-Livy is an open source REST interface for interacting with Spark from anywhere. It supports executing snippets of code or programs in a Spark context that runs locally or in YARN.
+# Livy Interpreter for Apache Zeppelin
+
+<div id="toc"></div>
+
+## Overview
+[Livy](http://livy.io/) is an open source REST interface for interacting with Spark from anywhere. It supports executing snippets of code or programs in a Spark context that runs locally or in YARN.
```

Comment:
@felixcheung Yeah right. Maybe next time we can update the available interpreters image in `index.md` :)